committees/project-structure.md

```diff
@@ -0,0 +1,55 @@
+# Rust Project Structure Committee
+
+## Mission
+
+The *Rust Project Structure* committee is tasked with determining the "shape" of the Rust Project and its membership,
+and to make proposals to the Council on any recommended changes,
+and to produce documentation on clarifying the Project structure.
+
+## Goals
+
+The goals of the committee are to address the following concerns, and to determine if there are any changes that should be proposed to the Council:
+
+- Review the structure of the Project. <https://github.com/rust-lang/leadership-council/issues/33>
+- Propose guidelines for determining whether or not to accept a new team in the project.
+  In essence, more clearly answer the question of "what is the Rust Project?".
+- Make recommendations on organizational cleanup.
+    - Identify concerns around the current structural schemes.
+    - Ensure all teams and other governance structures are "attached" to appropriate places in the structure. This includes working with teams to find appropriate homes, and ensuring such changes are ultimately reflected in the team metadata repository.
+    - Re-organize the top-level teams. <https://github.com/rust-lang/leadership-council/issues/32>
+    - Find homes for teams in the launching-pad.
```

Comment:
Perhaps in addition to this, we should clarify the long term role of the launching pad. I believe the Council RFC leaves it an open question whether the launching-pad should be a permanent fixture or whether it should be superseded by some other policy of where we fit hard-to-place teams in the over all structure. 

cc @thejpster

Reply:
It was made clear to me when applying for the role that the intention was for it to be temporary. If we decide there is a need for a top-level team as a container for other Organisational Units, it should probably have a new name.